vector/src/main/java/im/vector/app/features/voicebroadcast/listening/VoiceBroadcastPlayerImpl.kt

```diff
@@ -270,6 +282,8 @@ class VoiceBroadcastPlayerImpl @Inject constructor(
                 playbackTracker.updatePausedAtPlaybackTime(voiceBroadcast.voiceBroadcastId, positionMillis, positionMillis.toFloat() / duration)
             }
             playingState == State.Playing || playingState == State.Buffering -> {
+                stopPlayer()
```

Comment:
Is it right that when the playing state is `State.Playing` that we call `stopPlayer()`?

Reply:
Yes, as the user requested to listen from another position (potentially in another chunk file), we stop the current player and will start a new one just two lines later. If we don't stop it, the playback position which is saved in the next line could be erased by the current player

Reply:
... but you're right, this is not necessary, it is already done in `startPlayback` method. I'll remove it from this place